extensions/amp-bind/amp-bind.md

```diff
@@ -283,6 +283,7 @@ Only binding to the following components and attributes are allowed:
 | `<amp-carousel type=slides>` | `[slide]`<sup>1</sup> | Changes the currently displayed slide index. [See an example](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind).
 | `<amp-iframe>` | `[src]` | Changes the iframe's source URL. |
 | `<amp-img>` | `[alt]`<br>`[attribution]`<br>`[src]`<br>`[srcset]` | See corresponding [amp-img attributes](https://www.ampproject.org/docs/reference/components/media/amp-img#attributes). |
+| `<amp-list>` | `[src]` | Change the list's source URL. |
```

Comment:
How about "Fetches JSON from the new URL and re-renders while discarding old content."